lms/static/sass/features/_course-sock.scss

```diff
@@ -0,0 +1,172 @@
+.verification-sock {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  margin-top: $baseline;
+  max-width: 1180px;
```

Comment:
I'd like to see this be a variable. I don't know if it is hard-coded everywhere right now, or if there is one you can reuse.